QEfficient/generation/llm_generator.py

```diff
@@ -0,0 +1,298 @@
+# -----------------------------------------------------------------------------
+#
+# Copyright (c)  2024 Qualcomm Innovation Center, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# -----------------------------------------------------------------------------
+
+import json
+import os
+import torch
+import numpy as np
+import torch.nn as nn
+
+from typing import Dict, List, Optional, Union
+from threading import Thread
+
+from QEfficient.generation.cloud_infer import QAICInferenceSession
```

Comment:
this import should be below transformers import.
read more here: https://peps.python.org/pep-0008/#imports